README.md

```diff
@@ -30,7 +30,7 @@ variables continue to be accurate as data is updated.
     when our underlying assumptions are not true and will require us to
     address the issue before continuing.
 
-The [{pointblank}](https://rstudio.github.io/pointblank/) is
+The [{pointblank}](https://rich-iannone.github.io/pointblank/) is
```

Comment:
```suggestion
The [{pointblank}](https://rstudio.github.io/pointblank/) is
```

Posit officially sponsors pointblank for R and Python now. It's on Rstudio and Posit GHs respectively now.